### PR TITLE
net/mwan3: fixup parse json_load an null message

### PR DESCRIPTION
**Pull request by kysonlok**
@feckert 

If a interface is down, call ubus will return an null message which is
used to json_load, it causes json_get_vars gets value from last load and
l3_device various is nonzero.

Signed-off-by: Kyson Lok <kysonlok@gmail.com>

### CODE REVIEW

**Comment by kysonlok** [comment]
If a interface is down, l3_device is the same various.
```  
27394 root      1312 S    /bin/sh /usr/sbin/mwan3track wan eth0.2 unknown 192.168.12.102 8.8.4.4 8.8.8.8 208.67.222.222 208.67.220.
27749 root      1316 S    /bin/sh /usr/sbin/mwan3track wwan eth0.2 unknown 0.0.0.0 8.8.4.4 8.8.8.8 208.67.222.222 208.67.220.220
28130 root      1316 S    /bin/sh /usr/sbin/mwan3track tethering eth0.2 unknown 0.0.0.0 8.8.4.4 8.8.8.8 208.67.222.222 208.67.220.2
28468 root      1316 S    /bin/sh /usr/sbin/mwan3track modem eth0.2 unknown 0.0.0.0 8.8.4.4 8.8.8.8 208.67.222.222 208.67.220.220
```

File: net/mwan3/files/usr/sbin/mwan3 [hunk 1]
```diff
@@ -61,13 +61,18 @@ ifup()
...
+	[ -n "$status" ] && {
+		json_load $status
+		json_get_vars up l3_device 
+	}
+
 	config_get enabled "$1" enabled 0
 
-	if [ "$up" -eq 1 ] \
+
+	if [ "$up" == "1" ] \
```
Comment (feckert) [inline net/mwan3/files/usr/sbin/mwan3]: Why did you change the check from '-eq' to '=='?

File: net/mwan3/files/usr/sbin/mwan3 [hunk 2]
```diff
@@ -61,13 +61,18 @@ ifup()
...
 		&& [ -n "$l3_device" ] \
-		&& [ "$enabled" -eq 1 ]; then
+		&& [ "$enabled" == "1" ]; then
```
Comment (feckert) [inline net/mwan3/files/usr/sbin/mwan3]: The same her
Why did you change the check from '-eq' to '=='?

**Comment by feckert** [comment]
Thanks for the hint.
Could you also update the **PKG_VERSION**

File: net/mwan3/files/usr/sbin/mwan3 (hunk 1 above)
Comment (kysonlok) [reply net/mwan3/files/usr/sbin/mwan3]: It compares string type with integer type, if the length of string is zero, it causes *sh: out of rang* error on shell script.

File: net/mwan3/files/usr/sbin/mwan3 (hunk 2 above)
Comment (kysonlok) [reply net/mwan3/files/usr/sbin/mwan3]: The various enabled with double quotes is string type, while eq is compared with two integer, I think  that using '==' is more reasonable for comparison two string type, otherwise, if the length of string is zero, it causes *out of range* error.

**Comment by kysonlok** [comment]
Yes, if you configure the interface on /etc/config/network, using ubus can get valid json content, but if it missing on /etc/config/network, while configuring on /etc/config/mwan3, it will cause mwan3track a nonexist interface. So it is better to deal with fault tolerant, right?

File: net/mwan3/files/usr/sbin/mwan3 (hunk 2 above)
Comment (kysonlok) [reply net/mwan3/files/usr/sbin/mwan3]: @feckert  Yes, we change *==* to *=*.

File: net/mwan3/files/usr/sbin/mwan3 [hunk 3]
```diff
@@ -61,13 +61,18 @@ ifup()
...
+	status=$(ubus -S call network.interface.$1 status)
+	[ -n "$status" ] && {
+		json_load $status
+		json_get_vars up l3_device 
+	}
+
 	config_get enabled "$1" enabled 0
 
-	if [ "$up" -eq 1 ] \
+
```
Comment (feckert) [inline net/mwan3/files/usr/sbin/mwan3]: Could you please remove this new line?

**Comment by feckert** [comment]
@kysonlok sorry for the delay
could you please make the remaining changes
*  '==' vs. '='
* remove the new line in net/mwan3/files/usr/sbin/mwan3 on 37
* and change the PKG_VERSION from **2.7.0** to **2.7.1**

I have to review and test first https://github.com/openwrt/packages/pull/6515 and then we can merge your PR

**Comment by kysonlok** [comment]
@feckert  Well, I have changed it. Pls have a review.

**Comment by feckert** [comment]
@hnyman since the last PR 
https://github.com/openwrt/packages/pull/6515
was merged we could now merge this one as well?
Could you also rebase the PKG_VERSION 2.7.0 to 2.7.1?

**Comment by feckert** [comment]
@kysonlok could you please rebase your changes again so we could finally merge your changes?
